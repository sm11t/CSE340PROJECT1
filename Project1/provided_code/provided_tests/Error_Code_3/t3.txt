TASKS
    1 3 2
POLY
    F(Y,Z) = Y + Z;
    G = x - 1;
EXECUTE
    INPUT A;
    w = 
    F(
        G2(
            F(
                C(C(C(3))),
                G(A)
            )
        ),
        K(F(2,
            B(A)
        ))
    );
    OUTPUT w;
INPUTS
    3 4 5