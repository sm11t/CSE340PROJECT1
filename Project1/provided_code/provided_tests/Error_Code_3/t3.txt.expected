Semantic Error Code 3: 10 12 12 12 16 17 
